### PR TITLE
EP-163: Create new properties

### PR DESCRIPTION
**Pull request by Arkariang**
# 📲 What
- created user property `user_is_admin`
- created project property `project_has_add_ons`
- deleted no longer necessary Koala files

# 👀 See
new_properties

|  |  |

# 📋 QA

- Log In with an admin user
- See a project page to a project with addons ie: [Ares Expedition - The Terraforming Mars Card Game](https://staging.kickstarter.com/projects/strongholdgames/ares-expedition-the-terraforming-mars-card-game?ref=android_project_share)

# Story 📖

[EP-163](https://kickstarter.atlassian.net/browse/EP-163)

### CODE REVIEW

File: app/src/main/java/com/kickstarter/ui/views/KSWebView.kt [hunk 1]
```diff
@@ -127,7 +125,7 @@ class KSWebView : FrameLayout, KSWebViewClient.Delegate {
     }
 
     fun evaluateJavascript(javascript: String?, resultCallback: ValueCallback<String>?) {
-        internal_web_view.evaluateJavascript(javascript, resultCallback)
+        javascript?.let { internal_web_view.evaluateJavascript(it, resultCallback) }
```
Comment (leighdouglas) [inline app/src/main/java/com/kickstarter/ui/views/KSWebView.kt:128]: Is this change related to adding these new properties?

Comment (Arkariang) [reply app/src/main/java/com/kickstarter/ui/views/KSWebView.kt:128]: not really, I just saw it and fixed it

File: app/src/main/java/com/kickstarter/libs/utils/AnalyticEventsUtils.kt [hunk 2]
```diff
@@ -102,6 +102,7 @@ object AnalyticEventsUtils {
     fun userProperties(user: User, prefix: String = "user_"): Map<String, Any> {
         val properties = HashMap<String, Any>()
         properties["uid"] = user.id()
+        properties["is_admin"] = user.isAdmin ?: false
```
Comment (leighdouglas) [inline app/src/main/java/com/kickstarter/libs/utils/AnalyticEventsUtils.kt:105]: if `user.isAdmin` is a boolean, i dont think we need the elvis operator. if not then disregard

**Review by leighdouglas — APPROVED** [review]
Looks great! 🎉
